packages/lesswrong/lib/collections/comments/views.js

```diff
@@ -186,3 +186,11 @@ ensureIndex(Comments, {legacyId: "hashed"});
 
 // Used in scoring cron job
 ensureIndex(Comments, {inactive:1,postedAt:1});
+
+Comments.addView('repliesToAnswer', function (terms) {
+  return {
+    selector: {answerId: terms.answerId},
+    options: {sort: {baseScore: -1}}
+  };
+});
+ensureIndex(Comments, {answerId:1, baseScore:-1});
```

Comment:
Merge with current devel and wrap this in `augmentForDefaultView`; otherwise you'll end up with a full index scan.